labshare-client/src/components/SearchForm.vue

```diff
@@ -1,12 +1,36 @@
 <i18n>
-{
-  "en": {
-    
-  },
-  "de": {
-    
-  }
-}
+    {
+    "en": {
+    "text1": "Looking for",
```

Comment:
The keys here are unacceptable. They should describe what they are for so anyone can see at a glance where they should go and what we intend for them. Once that is done, the texts in titles (e.g., field labels and headers) should be made in title case